tests/schemas/api_v3/invoke.schema.js

```diff
@@ -21,7 +21,7 @@ const invokeMetaResponseSchema = {
 };
 
 const errorSchema = {
-	message: Joi.string().required(),
+	data: Joi.object({ error: Joi.string().required() }).required(),
```

Comment:
Did we modify any schemas?

Reply:
Incase of validation exception (BAD REQUEST) we return error inside data 
https://github.com/LiskHQ/lisk-service/blob/development/services/blockchain-indexer/methods/dataService/controllers/invoke.js#L39